tests/test_configuration_common.py

```diff
@@ -66,9 +66,16 @@ def create_and_test_config_with_num_labels(self):
         self.parent.assertEqual(len(config.id2label), 3)
         self.parent.assertEqual(len(config.label2id), 3)
 
+    def check_config_can_be_init_without_params(self):
+        if self.config_class.is_composition:
+            return
+        config = self.config_class()
```

Comment:
This makes sure that every config can be instantiated without providing any parameters.